src/PersonObjects/Sleeve/Work/SleeveBladeburnerWork.ts

```diff
@@ -82,6 +84,7 @@ export class SleeveBladeburnerWork extends SleeveWorkClass {
       type: SleeveWorkType.BLADEBURNER as "BLADEBURNER",
       actionType: this.actionType,
       actionName: this.actionName,
+      tasksCompleted: this.tasksCompleted,
```

Comment:
Since this property is being added to the API copy for Sleeve Blade and Crime work, it should also be added in ScriptEditor/NetscriptDefinitions.d.ts definitions for SleeveCrimeTask and SleeveBladeburnerTask

Reply:
pretty sure I just submitted what you asked for

Reply:
Yep! Will still need to re-run `npm run doc` so the generated docs get updated too

Reply:
It's like that. You won't see any result from the command-like output or from `git diff`, because it adds the changes as staged. You'll see them if you do `git status` or other commands like `git commit `.

Reply:
thanks.  I wanted to make sure I didn't have something mis-configured.  The messages about the 'fail to fine node.js install directory' made me think something wasn't working properly.